test/e2e/autoscale.go

```diff
@@ -54,7 +54,8 @@ const (
 	autoscaleTestImageName = "autoscale"
 )
 
-type testContext struct {
+// TestContext includes context for autoscaler testing.
+type TestContext struct {
```

Comment:
interestingly enough you don't need to make this public, but it'll annoy govet, I think :)

Reply:
yeah, Golint complains if if it is not public.